spec/controllers/api/v2/api_works_spec.rb

```diff
@@ -464,6 +473,8 @@
       after(:all) do
         @work&.destroy
         WebMock.reset!
+        @archivist.destroy
```

Comment:
Can you switch from `before(:all)`/`after(:all)` to `before`/`after`, which cleans up database records after each test, instead of manually destroying each of them at the end?

Reply:
Done. I didn't switch it in my first commit because it means that the test time increases. On my local setup running `spec/controllers/api/v2/api_works_spec.rb` in a fresh container now takes 22s instead of 11s.

Reply:
Thanks! Yeah, the WebMock setup can likely be optimized a little. But this should address the leaked state problem.